bot/exts/evergreen/emoji_count.py

```diff
@@ -17,73 +18,67 @@ class EmojiCount(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    def embed_builder(self, emoji: dict) -> discord.Embed:
+    def embed_builder(self, emoji: dict) -> [discord.Embed, str]:
         """Generates an embed with the emoji names and count."""
         embed = discord.Embed(
             color=Colours.orange,
             title="Emoji Count",
             timestamp=datetime.datetime.utcnow()
         )
+        msg = []
 
         if len(emoji) == 1:
             for key, value in emoji.items():
-                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
+                msg.append(f"There are **{len(value)}** emojis in the **{key}** category")
                 embed.set_thumbnail(url=random.choice(value).url)
+
         else:
-            msg = ''
             for key, value in emoji.items():
                 emoji_choice = random.choice(value)
                 emoji_info = f'There are **{len(value)}** emojis in the **{key}** category\n'
-                msg += f'<:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}'
-            embed.description = msg
-        return embed
+                if emoji_choice.animated:
+                    msg.append(f'<a:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}')
+                else:
+                    msg.append(f'<:{emoji_choice.name}:{emoji_choice.id}> {emoji_info}')
+        return embed, msg
 
     @staticmethod
-    def generate_invalid_embed(ctx: commands.Context) -> discord.Embed:
+    def generate_invalid_embed(ctx: commands.Context) -> [discord.Embed, str]:
         """Genrates error embed."""
```

Comment:
Should be `generates`
```suggestion
        """Generates error embed."""
```